distribution/pom.xml

```diff
@@ -9,6 +9,8 @@
 
   <artifactId>streamx-cli-distribution</artifactId>
 
+  <name>StreamX Cli : DISTRIBUTION</name>
```

Comment:
My bad. I meant not uppercase but capitalised...

So here: `StreamX Cli : Distribution`